addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4764,6 +4764,31 @@ registry.DynamicSvg = SnippetOptionWidget.extend({
     _computeVisibility(methodName, params) {
         return this.$target.is("img[src^='/web_editor/shape/']");
     },
+    /**
+     * Returns the default colors for the currently selected SVG.
+     *
+     * @private
+     */
+    _getDefaultColors() {
+        const svgSrc = this.$target.attr("src");
+        if (!svgSrc) {
+            return {};
+        }
+        const url = new URL(svgSrc, window.location.origin);
+        return Object.fromEntries(url.searchParams.entries());
```

Comment:
not sure to see the color returned ? seems like it return all searchparam

Reply:
Moving the filtering from _renderCustomXml (below) to this method